deployment.go

```diff
@@ -127,7 +165,8 @@ var deploymentStatusCmd = &cobra.Command{
 
 		switch resp.StatusCode {
 		case http.StatusOK:
-			fmt.Printf("(Status code %d) %s\n", resp.StatusCode, body)
+			fmt.Printf("(Status code %d) Daemon at remote '%s' online at %s\n", resp.StatusCode,remotename, host)
+			fmt.Printf(" %s", body)
```

Comment:
Probably want to remove the space before `%s` here - might make formatting a bit weird